### PR TITLE
ElisaKendall/FBC-226

### PR DESCRIPTION
**Pull request by ElisaKendall**

### CODE REVIEW

File: AboutFIBOProd.rdf [hunk 1]
```diff
@@ -48,10 +48,30 @@
     <owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FND/AgentsAndPeople/Agents/"/>
     <owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/SEC/Securities/SecuritiesIdentification/"/>
     <owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FBC/FunctionalEntities/BusinessCentersIndividuals/"/>
+<<<<<<< HEAD
```
Comment (trypuz) [inline AboutFIBOProd.rdf]: To remove

File: AboutFIBOProd.rdf [hunk 2]
```diff
@@ -48,10 +48,30 @@
...
+    <owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FBC/FunctionalEntities/NorthAmericanEntities/CARegulatoryAgencies/"/>
+	<owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FBC/FunctionalEntities/NorthAmericanEntities/USExampleIndividuals/"/>
+    <owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FBC/FunctionalEntities/NorthAmericanEntities/USFinancialServicesEntities/"/>
+	<owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FBC/FunctionalEntities/NorthAmericanEntities/USFinancialServicesEntitiesIndividuals/"/>
+	<owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FBC/FunctionalEntities/NorthAmericanEntities/USPostalServiceAndAddressing/"/>
+    <owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FBC/FunctionalEntities/NorthAmericanEntities/USPostalServiceAndAddressingIndividuals/"/>
+	<owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FBC/FunctionalEntities/NorthAmericanEntities/USRegulatoryAgencies/"/>
+=======
```
Comment (trypuz) [inline AboutFIBOProd.rdf]: To remove

File: AboutFIBOProd.rdf [hunk 3]
```diff
@@ -48,10 +48,30 @@
...
     <owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FBC/FunctionalEntities/NorthAmericanEntities/CAFinancialServicesEntities/"/>
     <owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/BE/OwnershipAndControl/CorporateControl/"/>
     <owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FBC/ProductsAndServices/FinancialProductsAndServices/"/>
     <owl:imports rdf:resource="https://spec.edmcouncil.org/fibo/ontology/FBC/FunctionalEntities/NorthAmericanEntities/USFinancialServicesEntitiesIndividuals/"/>
+>>>>>>> 19463d570b1680a243fd71717a995b25e6f6bb0b
```
Comment (trypuz) [inline AboutFIBOProd.rdf]: To remove

File: AboutFIBOProd.rdf (hunk 1 above)
Comment (rivettp) [reply AboutFIBOProd.rdf]: @trypuz I'm finding it hard to understand what GitHub's telling me about what you want @ElisaKendall to change. Can you please describe the changes you want and why.
Note that the reason this has been held up (see comments from me and Mike) is that this is expanding into a totally new area that seems outside the scope of FIBO and not supporting any documented FIBO use case. And needs a more general presentation and architectural discussion which would have been with the FLT and now I guess the Steering Team (?) which has not been constituted yet.

Comment (trypuz) [reply AboutFIBOProd.rdf]: It's about line 51.

Comment (ElisaKendall) [reply AboutFIBOProd.rdf]: The process for generating the AboutFIBOProd and AboutFIBODev ontologies appears to be broken.  I'll fix the conflict in this pull request, but at a higher level, the broader issue needs to be addressed as well.  Having an ontology IRI that is "file:///AboutFIBO" is wrong for so many reasons, and there is no metdata at all in these files.

**Pull request closed without merging**